README.md

```diff
@@ -8,48 +8,70 @@
 | Frontend Dapp      | Yes                           |
 | Wallet Integration | Yes                           |
 
-## Quickstart
-
-In a rush? You can follow the steps below to get up and running immediatley:
-
-1. `taq scaffold https://github.com/ecadlabs/taqueria-scaffold-taco-shop taco-shop`
-2. `npm run setup`
```

Comment:
I think we should restore this, and add a third step:
`npm run start:app`

Then add a new script entry to the root package.json that looks like this:
`"start:app": "cd app && npm run start"`

Then the react is running in a dev server.

Reply:
Added.